python-nikola-git/PKGBUILD

```diff
@@ -1,33 +1,93 @@
+# Maintainer: shmilee <echo c2htaWxlZS56anVAZ21haWwuY29tCg==|base64 -d>
 # Maintainer: Chris Warrick <aur@chriswarrick.com>
-pkgname=python-nikola-git
+pkgbase=python-nikola-git
 _pyname=nikola
 _gitname=nikola
-pkgver=7.4.1.r0.g64e8e4a
+pkgname=('python-nikola-git' 'python2-nikola-git' 'python-nikola-doc-git')
+pkgver=7.4.1.r45.g34f02ad
 pkgrel=1
 pkgdesc='A modular, fast, simple, static website generator.  (git version)'
 arch=('any')
 url='http://getnikola.com/'
 license=('custom')
-depends=('python' 'python-doit' 'python-pygments' 'python-pillow'
-         'python-docutils' 'python-mako' 'python-unidecode' 'python-lxml'
-         'python-yapsy' 'python-pyrss2gen' 'python-dateutil' 'python-logbook'
-         'python-blinker' 'python-colorama' 'python-natsort')
-makedepends=('git')
 options=(!emptydirs)
-provides=('python-nikola')
-conflicts=('python-nikola' 'python2-nikola' 'python2-nikola-git')
+makedepends=('git' 'python' 'python-setuptools' 'python2' 'python2-setuptools' 'python-sphinx')
 source=("git://github.com/getnikola/${_gitname}.git")
 md5sums=('SKIP')
 
-package() {
+pkgver() {
   cd "${srcdir}/${_gitname}"
-  python3 setup.py install --root="${pkgdir}/" --optimize=1
-  install -D -m644 LICENSE.txt "${pkgdir}/usr/share/licenses/${pkgname}/LICENSE"
+  git describe --long | sed -E 's/([^-]*-g)/r\1/;s/-/./g;s/^v//g'
 }
 
-pkgver() {
+build() {
   cd "${srcdir}/${_gitname}"
-  git describe --long | sed -E 's/([^-]*-g)/r\1/;s/-/./g;s/^v//g'
+  cp -R "${srcdir}/${_gitname}" "${srcdir}/${_gitname}2"
+}
+
+package_python-nikola-git() {
+  depends=('python' 'python-doit' 'python-pygments' 'python-pillow'
+           'python-docutils' 'python-mako' 'python-unidecode' 'python-lxml'
+           'python-yapsy' 'python-pyrss2gen' 'python-dateutil' 'python-logbook'
+           'python-blinker' 'python-natsort')
+  depends+=('python-nikola-doc-git')
+  optdepends=('python-requests: for theme and plugin installation, and various web services'
+              'python-markdown: for Markdown support'
+              'python-jinja: for Jinja2 themes'
+              'python-livereload==2.3.2: for automatically refreshing browsers'
+              'python-pyphen>=0.9.1: for hyphenation'
+              'python-pygal==1.7.0: for SVG graph plotting'
+              'python-typogrify>=2.0.4: for typographical enhancements'
+              'python-webassets: for bundling assets'
+              'ipython-notebook: for ipynb support'
+              'ipython: for ipynb support'
+              'python-ghp-import>=0.4.1: for uploading to GitHub Pages'
+              'python-micawber: for embedding media in posts'
+              'python-phpserialize: for WordPress imports')
+  conflicts=('python-nikola')
+  cd "${srcdir}/${_gitname}"
+  python3 setup.py install --prefix=/usr --root="${pkgdir}" --optimize=1
+  ln -s ${_pyname} "$pkgdir/usr/bin/${_pyname}3"
+  rm -rf "$pkgdir/usr/share"
```

Comment:
use `${pkgdir}` unless you liked https://github.com/MrMEEE/bumblebee-Old-and-abbandoned/commit/a047be85247755cdbe0acce6f1dafc8beb84f2ac